engine-pinia/src/shims-vue.d.ts

```diff
@@ -1,5 +1,33 @@
 declare module "*.vue" {
   import { defineComponent } from "vue";
+
+  // For reasons unclear to me, the docstring for the WWTComponent component
```

Comment:
This is probably https://github.com/TypeStrong/typedoc/issues/2283, which I haven't gotten to yet...